### ISSUE TITLE
Issue in scores when training HEADER model for custom training data

### ISSUE THREAD

**Issue by Tanmay98** [kermitt2/grobid#930]
Hi,

- I tried training fulltext and segmentation model for my own custom data and it worked like a charm but when I tried training for the header model, I found some interesting observations.
- After running `./gradlew train_header` on 3 training files from both the default dataset  as well as my own custom dataset, the training paramters as well as the scores produced are vastly different.

### Grobid sample header xml

<img src="https://user-images.githubusercontent.com/31308303/176833967-5a436df2-20a7-4c63-85d7-3283a89e4457.png" width="70%" height="50%">
<p float="left">
  <img src="https://user-images.githubusercontent.com/31308303/176834474-b1077f0d-d644-418a-be3c-b6dfe40a0bd2.png" width="25%" />
  <img src="https://user-images.githubusercontent.com/31308303/176834584-4c837693-2b1a-48a0-a557-14d3a687ac16.png" width="50%" /> 
</p>

### My cutom header xml
<img src="https://user-images.githubusercontent.com/31308303/176834860-3d903cca-a76a-4232-a3f8-cf8216a65511.png" width="70%" height="50%">
<p float="left">
 <img src="https://user-images.githubusercontent.com/31308303/176835130-5c146c6b-e582-45cd-98a7-72eab389015a.png" width="25%" />
<img src="https://user-images.githubusercontent.com/31308303/176835197-c3bf1768-af74-48ee-be4a-6a2041bf0864.png" width="50%"/> 
</p>

- In above experiment, 3 training files were used for both cases. but the precision for **title tag** and **author tag** came out to be zero for my model.
- Can someone tell me why this might be happening? Thanks in advance!

**Comment by kermitt2** [comment]
Hello @Tanmay98 !

It seems that you train the model with only 3 headers - with so few examples the scores are totally random and to be expected.

The current set for the headers has around 650 annotated headers, and this is already a low amount given the learning rate of the model. I don't think you could obtain a basic usable model with less than 50-100 header training examples.

**Comment by Tanmay98** [comment]
Hi @kermitt2 ,
Thanks for your response. I agree the score is just random for a few files to train upon.

So, I'll update the results once I train header model for around 100 files of my custom data.

**Comment by Tanmay98** [comment]
Hi @kermitt2 ,
It's working when I trained for around50 files.

Thanks a lot!
